cms/djangoapps/contentstore/views/course.py

```diff
@@ -1229,21 +1243,21 @@ def get_user_partition(self):
     @staticmethod
     def get_usage_info(course, store):
         """
-        Get usage information for all Group Configurations.
+        Get usage information for all Group Configurations that can be referenced by split_test instances.
```

Comment:
This documentation is actually incorrect. This method appends "usage info" to all group configurations currently in use by any split_test instance. I'll update it.